publish_trips.py

```diff
@@ -161,6 +174,10 @@ def handle_data(csv_text):
     data = classify_bike_type(data)
     # Add month/year columns
     data = populate_month_year(data)
+    # Add datetime column
+    data = datetime_creation(data)
+    # Remove trips that are less than 2 minutes in length
+    data = [d for d in data if int(d.get("trip_duration_minutes", 0)) > 1]
```

Comment:
Hunter has been adding this filter manually to the data he provides to us but it is good practice to make sure we drop them to keep things consistent.